AjobK/Athena
# 207 storing images

closes #223, #215 #211 

 **What is it supposed to do**
You should be able to upload an image
You should be able to crop an image
You should be able to update an image
You shouldn't be able to upload any other file then  a .png, .jpg

 **How can we test your branch**
- reset the database
- register a user
- a default image should be displayed on the profile
- click the image
- crop the image
- click save
- now the profile picture should change
- logout
- refresh
- profile picture should be the same
- register a new user, and it should have the default picture

 **What browser have been checked**
